templates/tftmpl/condition.go

```diff
@@ -0,0 +1,54 @@
+package tftmpl
+
+import (
+	"fmt"
+	"io"
+	"log"
+)
+
+// Condition handles appending a run condition's templating to the tfvars.tmpl
+// file
+type Condition interface {
+	appendTemplate(io.Writer) error
+}
+
+var _ Condition = (*ServicesCondition)(nil)
+
+// ServicesCondition handles appending templating for the services run condition
+// This is the default run condition
+type ServicesCondition struct{}
+
+func (c *ServicesCondition) appendTemplate(w io.Writer) error {
+	// no-op: services conditon currently requires no additional condition
+	// templating. it relies on the monitoring template as the run condition
+	return nil
+}
+
+var _ Condition = (*CatalogServicesCondition)(nil)
```

Comment:
Can these be grouped at the top of the file near the interface? It helps me quickly identify what classes are in file that satisfy `Condition`

```go
var (
  _ Condition = (*...)
  _ Condition = (*...)
)
```

Reply:
Sure thing!